tests/api.spec.ts

```diff
@@ -77,6 +79,30 @@ describe("api unit test", () => {
         
     });
 
+    it("loadActions", async () => {
+        const allExtensioms = [{
+            packageJSON: {
+                "BASContributes": {
+                    "actions": [{
+                        "id" : "abc123",
+                        "actionType" : "COMMAND"
+                    }]
+                },
+            }
+        }]
+        _.set(vscode, "extensions.all", allExtensioms);
+
+        const action: IAction = {
+			"id" : "abc123",
+			"actionType" : ActionType.Command,
+		}
```

Comment:
const action: IAction = {
			"id" : "abc123",
			"actionType" : ActionType.Command,
		}
        const allExtensioms = [{
            packageJSON: {
                "BASContributes": {
                    "actions": [action]
                },
            }
        }]
        _.set(vscode, "extensions.all", allExtensioms);

Reply:
you have the vscode import for this set

Reply:
allExtensions should resemble package.json, so it should contain strings

Reply:
allExtension should resemble package.json, so it should only contain strings